assets/scss/_custom.scss

```diff
@@ -313,6 +323,11 @@ footer {
   }
 }
 
+footer {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.527),rgba(0, 0, 0, 0.5)) , url("/images/texture.png");
+  color: #e9e9e9;
+}
```

Comment:
where does this come from?

Reply:
This is my stylistic choice. It styles the site footer without any layout-level override; it's a blend of on our look before we used Docsy, and Docsy's own look (solid dark grey background to the footer).

If we drop using `texture.png`, we can change the page header and footer at the same time. That'd be a future PR.